src/main/java/org/apache/maven/plugins/javadoc/AbstractJavadocMojo.java

```diff
@@ -2393,7 +2393,7 @@ protected void executeReport( Locale unusedLocale )
         List<Path> modulePaths = new LinkedList<>();
         for ( String module :  aggregatedProject.getModules() )
         {
-            modulePaths.add( new File( aggregatedProject.getBasedir(), module ).toPath() );
+            modulePaths.add( new File( aggregatedProject.getBasedir(), module ).toPath().normalize() );

```

Comment:
You can probably use Paths.get here and avoid the File class completely.

Reply:
Have simpified code to use only class Path and not File.
Added also a regression test for the issue